### DISCUSSION TITLE
Bahas Ionic Framework

### DISCUSSION THREAD

**Discussion by bahyusanciko** [ngobrolin/web#37] [Ideas]
Apa Masih relevan pwa jadi native mobile ?

**Comment by KhenCahyo13** [comment]
Permisi mau jawab, sebelumnya CMIIW. Kalau bahas Ionic, Ionic bukan PWA, melainkan Web View. Web View sama PWA sendiri memang hampir mirip secara hasil, tapi mereka beda konsep. Soal relevan atau nggak ya relevan aja, kembali lagi tergantung kebutuhan, sekarang di indo lagi rame pake cross-platform seperti Flutter atau React Native, karena memang SDM nya rata2 JS dripada Java atau Kotlin yang bener2 Native. Tapi kalau ngomongin performance jelas masih cepet yang bener2 native dripada cross-platform. Kalau ngomongin framework JS buat mobile, saya sendiri lebih suka React Native dripada Ionic, soal performance masih React Native karna React Native bukan Web View seperti Ionic, tapi dia punya bridge sendiri buat ke native, hampir mirip Flutter yang punya bridge sendiri buat ke native, apalagi UI, masih native an React Native, jadi menurut saya bisa move ke React Native aja.

**Comment by bahyusanciko** [comment]
bahas kenapa masi hype di luar sana, karena development lebih cepet dari harus belajar native code nya belajar pola pikir mobilenya si
